paymentsheet/src/main/java/com/stripe/android/paymentelement/EmbeddedPaymentElementKtx.kt

```diff
@@ -68,4 +56,25 @@ fun rememberEmbeddedPaymentElement(
             resultCallback = onResult,
         )
     }
+
+    val callbacks = remember(builder) {
+        @OptIn(ExperimentalCustomPaymentMethodsApi::class, ExperimentalAnalyticEventCallbackApi::class)
+        PaymentElementCallbacks.Builder()
+            .createIntentCallback(builder.createIntentCallback)
+            .confirmCustomPaymentMethodCallback(builder.confirmCustomPaymentMethodCallback)
+            .externalPaymentMethodConfirmHandler(builder.externalPaymentMethodConfirmHandler)
+            .analyticEventCallback(builder.analyticEventCallback)
+            .rowSelectionImmediateActionCallback(builder.rowSelectionBehavior, embeddedPaymentElement)
+            .build()
+    }
+
+    if (
+        builder.rowSelectionBehavior is EmbeddedPaymentElement.RowSelectionBehavior.ImmediateAction
+    ) {
+        embeddedPaymentElement.clearPaymentOption()
```

Comment:
I don't think we can do this here, right? We only want to do this during configuration, not initialization.